pdpserver/server/log_mgr.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+type LevelCheckable interface {
+	GetLevel() logrus.Level
+}
+
+func GetLogLevel(logger logrus.FieldLogger) logrus.Level {
+	if lgr, ok := logger.(*logrus.Logger); ok {
+		return lgr.Level
+	} else if lgr, ok := logger.(LevelCheckable); ok {
+		return lgr.GetLevel()
+	}
+	// else
```

Comment:
It's quite obvious that here we have default type so don't need to put the comment.